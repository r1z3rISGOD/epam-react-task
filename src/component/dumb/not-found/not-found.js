import React from 'react'
import './not-found.scss'

export const NotFound = () => {
  return (
        <div className='not-found'>
            <div>ERROR 404</div>
            <div>PAGE NOT FOUND</div>
        </div>
  )
}
