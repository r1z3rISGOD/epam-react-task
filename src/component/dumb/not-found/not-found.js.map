{"version":3,"file":"not-found.js","sourceRoot":"","sources":["not-found.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAC9B,4BAAyB;AAElB,IAAM,QAAQ,GAAc;IACjC,OAAO,CACD,6BAAK,SAAS,EAAC,WAAW;QACtB,6CAAoB;QACpB,kDAAyB,CACvB,CACX,CAAA;AACH,CAAC,CAAA;AAPY,QAAA,QAAQ,YAOpB"}