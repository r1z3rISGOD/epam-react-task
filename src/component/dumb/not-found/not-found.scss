@import "src/_variables/variables";

.not-found {
  width: $width;
  height: 100vh;
  background-color: white;
  position: absolute;
  font-size: 7rem;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  font-family: $font;
}
