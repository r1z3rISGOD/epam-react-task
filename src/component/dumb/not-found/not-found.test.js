import { NotFound } from './not-found.tsx'
import React from 'react'

it('not found', () => {
  const comp = shallow(<NotFound/>)
  expect(comp).toMatchSnapshot()
})
