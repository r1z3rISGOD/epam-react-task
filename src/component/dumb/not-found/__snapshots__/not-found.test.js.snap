// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not found 1`] = `
<div
  className="not-found"
>
  <div>
    ERROR 404
  </div>
  <div>
    PAGE NOT FOUND
  </div>
</div>
`;
